### DISCUSSION TITLE
Cast to smart screens like Echo Show or Home Hub

### DISCUSSION THREAD

**Discussion by fabioteixei** [mulder82/immich-slideshow#3] [Ideas]
It's possible to use this or even use the same backend to create an home assistant add-on so we can cast an Immich slideshow on Google Home Hub or Alexa Echo Show?

Thanks

**Comment by mulder82** [comment]
If Google Home Hub or Alexa Echo Show allows you to open any page in the browser then yes, just create a new home assistant dashboard with immich-slideshow card and configure the browser to display it.

**Reply by fabioteixei** [reply]
I will validate that later, I don't know if they are capable of showing html pages.

But I know that they can show images, home assistant can cast almost any image to theses devices. It's possible to change your code so it will pass the images url on the slideshow directly to Home Hub? I mean using some kind of automation.

I'm not an programmer, I work on IT but on the infra side of things, if not I would try myself to create an HA plugin.
